### ISSUE TITLE
bug: [Accessibility] MessageOptions should be accessible with focus

### ISSUE THREAD

**Issue by jkjustjoshing** [GetStream/stream-chat-react#2106]
**Describe the bug**

When using the default styles, the three dots "MessageOptions" component can only be made visible with a mouse hover. The options aren't accessible with only keyboard.

**To Reproduce**

1. Open channel using default stream-chat-react components
2. Try to navigate to the "MessageOptions" button next to a message using the keyboard only

**Expected behavior**

On the style that defines `.str-chat__li:hover`, it should also define `.str-chat__li:has(:focus-visible)` as a trigger for applying `display: flex`.

It could also use `.str-chat__li:focus-within` as a fallback for browsers that don't support `:has()`.

**Package version**
- stream-chat-react: 10.8.7
- stream-chat-css: ?
- stream-chat-js: 8.10.1

**Desktop (please complete the following information):**

- OS: macOS
- Browser: Chrome
- Version: Latest

**Smartphone (please complete the following information):**

**Additional context**

**Comment by MartinCupela** [comment]
@jkjustjoshing thank you for the proposal. We will take a look at this during the following days.

**Comment by myandrienko** [comment]
Fixed in https://github.com/GetStream/stream-chat-css/pull/272.
